core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/store/dot-experiments-configuration-store.ts

```diff
@@ -120,20 +149,47 @@ export class DotExperimentsConfigurationStore extends ComponentStore<DotExperime
                             this.updateTabTitle(experiment);
                         },
                         (error: HttpErrorResponse) => throwError(error),
-                        () => this.setComponentStatus(LoadingState.LOADED)
+                        () => this.setComponentStatus(ComponentStatus.IDLE)
                     )
                 )
             )
         );
     });
+    readonly startExperiment = this.effect((experiment$: Observable<DotExperiment>) => {
+        return experiment$.pipe(
+            tap(() => this.setComponentStatus(ComponentStatus.SAVING)),
+            switchMap((experiment) =>
+                this.dotExperimentsService.start(experiment.id).pipe(
+                    tapResponse(
+                        (response) => {
+                            this.messageService.add({
+                                severity: 'info',
+                                summary: this.dotMessageService.get(
+                                    'experiments.action.start.confirm-title'
+                                ),
+                                detail: this.dotMessageService.get(
+                                    'experiments.action.start.confirm-message',
+                                    experiment.name
+                                )
+                            });
+                            this.setExperiment(response);
+                        },
+                        (error) => throwError(error),
+                        () => this.setComponentStatus(ComponentStatus.IDLE)
+                    ),
+                    catchError(() => EMPTY)
```

Comment:
don't you need a way to handle errors?

Reply:
I have a ticket to fix that. Globally with an Interceptor, in that way, we don't need to put more code to handle errors in the file.

Reply:
Added `dotHttpErrorManagerService` while going global.